changelog.txt

```diff
@@ -1,12 +1,26 @@
 == Changelog ==
 
-= [5.1.6] TBD =
+
+= [TBD] =
+
+* Fix - Fixed cart calculation inconsistency with WooCommerce when the "Number of decimals" setting was set to `0`. [ETP-324]
+
+= [5.1.7] 2021-08-03 =
+
+* Feature - Added export button next to the page title on the Attendees page. [ET-1145]
+* Tweak - Changed the word `Purchaser` to `Attendee` when email confirmation is sent for purchasing a ticket. [ETP-655]
+* Tweak - Added `$attendees` parameter to the `tribe_report_page_after_text_label` action. [ET-1145]
+* Tweak - Removed the edit column when printing the Attendees list. [ETP-702]
+* Tweak - Added "Delete" functionality for the tickets area in the classic editor. [ET-1107]
+* Language - 1 new strings added, 61 updated, 0 fuzzied, and 0 obsoleted
+
+= [5.1.6] 2021-07-07 =
```

Comment:
is this from the previous release?  if so, all good, but we typically don't update the changelog, just in the readme.

Reply:
Yes, I have added the missing entries from the last couple of releases as well. 

I also thought that we should only update the readme but the PR template checklist above is updated saying that we need to update both.